x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/edit_package_policy_page/index.tsx

```diff
@@ -232,6 +230,8 @@ export const EditPackagePolicyForm = memo<{
                   ...basePolicyInputVars,
                 };
               }
+              // Fix duration vars, if it's a migrated setting, and it's a plain old number with no suffix
+              newVars = fixApmDurationVars(newVars);
```

Comment:
maybe there could be a check to only run this logic for apm package type

Reply:
Where I can check if it's an apm package type?

Reply:
here you can use basePackage: `basePackage.name === 'apm'`